solutions/standard/variables.tf

```diff
@@ -0,0 +1,190 @@
+  type        = string
+  description = "The IBM Cloud API key to deploy resources."
+  sensitive   = true
+}
+variable "use_existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "The name of a new or an existing resource group to provision the Databases for Redis in. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
+}
+
+variable "prefix" {
+  type        = string
+  description = "Prefix to add to all resources created by this solution."
+  default     = null
+}
+
+variable "name" {
+  type        = string
+  description = "The name of the Databases for Redis instance. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
+  default     = "redis"
+}
+
+variable "region" {
+  description = "The region where you want to deploy your instance."
+  type        = string
+  default     = "us-south"
+}
+
+variable "redis_version" {
+  description = "The version of the Databases for Redis instance. If no value is specified, the current preferred version of Databases for Redis is used."
+  type        = string
+  default     = null
+}
+
+variable "members" {
+  type        = number
+  description = "The number of members that are allocated. [Learn more](https://cloud.ibm.com/docs/databases-for-redis?topic=databases-for-redis-resources-scaling)."
+  default     = 2
+}
+
+variable "member_memory_mb" {
+  type        = number
+  description = "The memory per member that is allocated. [Learn more](https://cloud.ibm.com/docs/databases-for-redis?topic=databases-for-redis-resources-scaling)"
+  default     = 4096
+}
+
+variable "member_cpu_count" {
+  type        = number
+  description = "The dedicated CPU per member that is allocated. For shared CPU, set to 0. [Learn more](https://cloud.ibm.com/docs/databases-for-redis?topic=databases-for-redis-resources-scaling)."
+  default     = 0
+}
+
+variable "member_disk_mb" {
+  type        = number
+  description = "The disk that is allocated per member. [Learn more](https://cloud.ibm.com/docs/databases-for-redis?topic=databases-for-redis-resources-scaling)."
+  default     = 5120
+}
+
+variable "member_host_flavor" {
+  type        = string
+  description = "The host flavor per member. [Learn more](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/database#host_flavor)."
+  default     = "multitenant"
+}
+
+variable "configuration" {
+  description = "Database Configuration for Redis instance."
+  type = object({
+    maxmemory                   = optional(number)
+    maxmemory-policy            = optional(string)
+    appendonly                  = optional(string)
+    maxmemory-samples           = optional(number)
+    stop-writes-on-bgsave-error = optional(string)
+  })
+  default = null
+}
```

Comment:
This is new and may want back porting.

It is also complex and will want adding to the DA documentation. (see comment on users variable. The description should have a `[learn more]()` style link.

Reply:
addressed